testsuite/LaTeXToMathML/AMScd/issue1013.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>AMScd: close cells before reading arrows</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../header.js"></script>
+    <script type="text/javascript">
+      function preMathJax()
+      {
+      gConfigObject.TeX = { extensions: ["AMSmath.js", "AMScd.js"] };
+      }
```

Comment:
I've never liked Fred's style of indenting.  I would prefer the more standard

    function preMathJax() {
      gConfigObject.TeX = { extensions: ["AMSmath.js", "AMScd.js"] };
    }

but will merge anyway as there are lots of tests that are formatted this way.  I'm switching them over as I write new ones.